File: src/Mosaico.Domain.ProjectManagement/Models/CertificateGenerator/CertificateConfiguration.cs
namespace Mosaico.Domain.ProjectManagement.Models.CertificateGenerator
{
    public class CertificateConfiguration
    {
        public BaseTextBlock InvestorName { get; set; }
        public TokensAmountBaseBlock TokensAmount { get; set; }
        public BaseTextBlock Date { get; set; }
        public BaseTextBlock Code { get; set; }
        public LogoBlock LogoBlock { get; set; }
    }
}

File: frontend/mosaico-web-ui/projects/mosaico-project/src/lib/models/certificate-configuration.ts
import {TextBlock} from "./certificate/text-block";
import {TokensAmount} from "./certificate/tokens-amount";
import {LogoBlock} from "./certificate/logo-block";

export interface CertificateConfiguration {
    investorName: TextBlock;
    tokensAmount: TokensAmount;
    date: TextBlock;
    code: TextBlock;
    logoBlock: LogoBlock;
}
